### PR TITLE
Display articles from the same domain name 

### PR DESCRIPTION
**Pull request by nicosomb**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes/no
| Documentation | yes/no
| Translation   | yes/no
| CHANGELOG.md  | yes/no
| License       | MIT

Feature to display all the articles from the same domain name than a single article

### CODE REVIEW

File: src/Wallabag/CoreBundle/Controller/EntryController.php [hunk 1]
```diff
@@ -517,6 +517,24 @@ public function shareEntryAction(Entry $entry)
         );
     }
 
+    /**
+     * List the entries with the same domain as the current one.
+     *
+     * @param int $page
+     *
+     * @Route("/same-domain/{id}/{page}", requirements={"id" = ".+"}, defaults={"page" = 1}, name="same_domain")
```
Comment (tcitworld) [inline src/Wallabag/CoreBundle/Controller/EntryController.php:525]: Just `domain` would be fine.

File: src/Wallabag/CoreBundle/Controller/EntryController.php [hunk 2]
```diff
@@ -517,6 +517,24 @@ public function shareEntryAction(Entry $entry)
...
+     *
+     * @return \Symfony\Component\HttpFoundation\Response
+     */
+    public function getSameDomainEntries(Request $request, $page = 1)
+    {
+        if (!$this->get('craue_config')->get('share_public')) {
```
Comment (tcitworld) [inline src/Wallabag/CoreBundle/Controller/EntryController.php:531]: :thinking:

File: src/Wallabag/CoreBundle/Resources/translations/messages.cs.yml [hunk 3]
```diff
@@ -28,6 +28,7 @@ entry:
     page_titles:
         all: Všechny položky
         untagged: Položky bez štítků
+        # same_domain: Same domain
```
Comment (j0k3r) [inline src/Wallabag/CoreBundle/Resources/translations/messages.cs.yml:31]: You don't need to update all translation now. Just provide the English one and weblate will do the rest.

**Review by j0k3r — CHANGES_REQUESTED** [review]
What will be the rendering of the icon `language` for that feature?

File: src/Wallabag/CoreBundle/Resources/translations/messages.cs.yml (hunk 3 above)
Comment (nicosomb) [reply src/Wallabag/CoreBundle/Resources/translations/messages.cs.yml]: Fixed.

**Comment by nicosomb** [comment]
Capture d’écran_2021-01-04_09-31-58
Capture d’écran_2021-01-04_09-31-29

**Comment by j0k3r** [comment]
Could you rebase against the 2.5.0 branch (so GitHub Actions will be triggered)?

File: src/Wallabag/CoreBundle/Resources/views/themes/material/Entry/_card_actions.html.twig [hunk 4]
```diff
@@ -9,6 +9,7 @@
 
     <ul class="tools right">
         <li>
+            <a title="{{ 'entry.list.show_same_domain'|trans }}" class="tool grey-text" href="{{ path('same_domain', { 'id': entry.id }) }}"><i class="material-icons">language</i></a>
```
Comment (j0k3r) [inline src/Wallabag/CoreBundle/Resources/views/themes/material/Entry/_card_actions.html.twig:12]: That translation is missing in the translation file.

Comment (nicosomb) [reply src/Wallabag/CoreBundle/Resources/views/themes/material/Entry/_card_actions.html.twig:12]: Fixed. Good catch.

**Review by j0k3r — APPROVED** [review]

**Pull request merged**